### ISSUE TITLE
Wrong file generated

### ISSUE THREAD

**Issue by ftomassetti** [openvalidation/openvalidation#66]
If I run:

java -jar openvalidation.jar --culture en --language java \
     --rule "your age HAS to be greater than 22" --schema "{ age: 21 }" --output age_check.js
The file `age_check.java` is generated, therefore the output directive is not precisely respected, which is confusing to me

**Comment by thecodemonkey** [comment]
Yes, Federico, once again you're right. This parameter is not entirely clear.

The parameter --output(-o) actually defines the output directory. However, there is the parameter --single-file (-f) over it you can get all the generated code, which actually consists of 2 files: Implementation(Rule Set) and HUMLFramework in one file. If the -f switch is set, then you can specify the name of this single file with -o. I know that it is not really cleanly implemented at this point. And we should definitely change it.

https://docs.openvalidation.io/openvalidation-cli#single-file
https://docs.openvalidation.io/openvalidation-cli#output

My suggestion is:

1. That we use the --output (-o) parameter to enter only the output directory. (it is not allowed to give file names with extension. we can validate it.)
2. the parameter --single-file (-f) should no longer be a switch(boolean), but a string parameter to specify the output name of the single file. If you do not specify anything, the default name (HUMLValidator.xxx) will be used.

what do you think about this?

**Comment by ftomassetti** [comment]
it makes sense to me
